modules/userId/index.js

```diff
@@ -820,6 +820,13 @@ export function init(config) {
       auctionDelay = utils.isNumber(userSync.auctionDelay) ? userSync.auctionDelay : NO_AUCTION_DELAY;
       updateSubmodules();
     }
+    if (userSync && userSync.userIds && getUserIds().hasOwnProperty(userSync.ppid) && typeof getUserIds()[userSync.ppid] === 'string') {
```

Comment:
this won't work with every id provider, including id5, since the result of the `getUserIds()['id5id']` would be an object not a string.

the `eids` array is more standard so might be a better choice?